docs/cristin.ttl

```diff
@@ -514,14 +542,98 @@
     :mapstonvafield "entityDescription.contributor.role. only the first role of a person is considered." .
 
 :rollekode
-    a            rdf:Property ;
-    rdfs:comment """
+    a               :cristinfield ;
+    rdfs:comment    """
         The role of the respective Contributor for the respective publication affiliated for this role with the respective institution.
         This entry is immutable and does not change in time.
         """ ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :VARBEID_PERSON_STED_ROLLE_Class ;
+    :isignored      "false" ;
+    :mapstonvafield " maps to NVA Role" .
+
+:eierkode
+    a            :cristinfield ;
     rdfs:range   xsd:string ;
-    rdfs:domain  :VARBEID_PERSON_STED_ROLLE_Class ;
-    :isignored   "false" .
+    rdfs:domain  :VITENSKAPELIGARBEID_LOKAL_Class ;
+    rdfs:comment """The institution the currently owns the NVI result. Example UIO.
+    We do not know whether the values of this field are standardized and whether there is a defined set of values for this property's range""" ;
+    :isignored   "true" .
+
+
+:status_kontrollert
+    a            :cristinfield ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VITENSKAPELIGARBEID_LOKAL_Class ;
+    rdfs:comment """
+     It indicates if the respective post has been controlled or not. It receives values "J", "N".
+     It is unknown whether it refers to the Cristin entry or the NVI entry.
+    """ ;
+    :isignored   "true" .
+
+:brukernavn_kontrollert
+    a            :cristinfield ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VITENSKAPELIGARBEID_LOKAL_Class ;
+    rdfs:comment """
+     The username who controlled the respective entry. Unknown whether the field refers to the NVI entry or the Cristin entry.
```

Comment:
The username who controlled / judged / validated the respective entry *representing their respective institution*, distinguished by VITENSKAPELIGARBEID_LOKAL.eierkode.

Is contained in the VITENSKAPELIGARBEID_LOKAL object, so refers to the NVI processing record.